pandas/tests/groupby/test_groupby.py

```diff
@@ -2025,3 +2025,146 @@ def test_groupby_crash_on_nunique(axis):
         expected = expected.T
 
     tm.assert_frame_equal(result, expected)
+
+
+@pytest.mark.parametrize(
```

Comment:
can you make a new file, test_groupby_dropna

Reply:
yes, moved to `test_groupby_dropna`